src/main/java/thePackmaster/actions/evenoddpack/WindupAction.java

```diff
@@ -0,0 +1,34 @@
+package thePackmaster.actions.evenoddpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.ModifyDamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+
+public class WindupAction extends AbstractGameAction {
```

Comment:
I'll stop pointing them out. Basically if it's a one-off action and has less than 10 lines, please make it an anonymous action